src/components/Nav/Nav.js

```diff
@@ -23,27 +42,92 @@ function Nav() {
             );
           })}
         </div>
-        <N.InputBox placeholder="검색어 입력"></N.InputBox>
-        <div>
-          {MY_LIST.map(({ name }) => {
-            return <N.MyBtn key={name}>{name}</N.MyBtn>;
-          })}
-        </div>
-        <N.AddFeedBtn>글쓰기</N.AddFeedBtn>
+        <Search />
+        <N.MyBtnList>
+          {
+            // token === null &&
+            NONE_TOKEN_MY_LIST.map(({ name }, index) => {
+              return (
+                <N.MyBtn
+                  key={index}
+                  primary={name === '로그인'}
+                  onClick={() => {
+                    navigate('/login');
+                  }}
+                >
+                  {name}
+                </N.MyBtn>
+              );
+            })
+          }
```

Comment:
```suggestion
          <N.MyBtn primary={true} onClick={handleLogin}>
            {token ? '로그아웃' : '로그인'}
          </N.MyBtn>
```

token의 유무에 따라서 로그인 / 로그아웃 버튼을 다르게 보여주고 싶다면
이런 방식으로 하는 것이 더 좋을 것 같습니다!